demos/device_shadow_for_aws/shadow_demo_main.c

```diff
@@ -568,7 +568,7 @@ int RunDeviceShadowDemo( bool awsIotMqttMode,
                          void * pNetworkCredentialInfo,
                          const void * pNetworkInterface )
 {
-    int returnStatus = EXIT_SUCCESS;
+    int returnStatus = pdPASS;
```

Comment:
We should use `BaseType_t` as the type for `pdPASS` and `pdFAIL`
```suggestion
    BaseType_t xDemoStatus = pdPASS;
```